README.md

```diff
@@ -8,11 +8,11 @@ This project provides REST based web services for exposing fees related informat
 
 ## Getting Started
 
-This is SpringBoot+maven based java application. Please see the Jenkinsfile in root folder to see the build and deployment pipeline.
+This is SpringBoot based java application. Please see the Jenkinsfile in root folder to see the build and deployment pipeline.
 
 ### Prerequisites
 
-You will need jdk and maven installed on your machine or use mvnw to install the prerequisites
+You will need jdk installed on your machine.
 
 ### Installing
 1. Clone the repo to your machine using git clone git@git.reform.hmcts.net:fees-register/fees-register-app.git
```

Comment:
update url to https://github.com/hmcts/ccfr-fees-register-app.git

Reply:
sure

Reply:
Updated